src/main/java/chess/controller/WebRestController.java

```diff
@@ -0,0 +1,51 @@
+package chess.controller;
+
+import chess.dto.ChessGameDto;
+import chess.dto.GameRoomDto;
+import chess.dto.MoveDto;
+import chess.service.ChessService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/games")
+public class WebRestController {
+    private final ChessService chessService;
+
+    public WebRestController(ChessService chessService) {
+        this.chessService = chessService;
+    }
+
+    @PostMapping("/{gameId}/pieces")
+    public ResponseEntity<ChessGameDto> startGame(@PathVariable int gameId) {
```

Comment:
아래 코드를 참고해서 컨트롤러의 다른 메서드도 수정해봅시다.

```suggestion
    @GetMapping("/{gameId}/pieces")
    public ResponseEntity<ChessGameDto> restart(@PathVariable int gameId) {
        return ResponseEntity.ok(chessService.loadGame(gameId));
    }
```

Reply:
구구 말씀대로 안정성과 가독성을 겸비한 빌더 패턴을 사용했습니다!!

Reply:
네!! 가독성이 좋은 빌더패턴을 사용하겠습니다!!